File: app/src/main/java/com/karim/booksapp/data/models/pojos/AccessInfo.java

package com.karim.booksapp.data.models.pojos;

import com.squareup.moshi.Json;

public class AccessInfo {

    @Json(name = "country")
    public String country;
    @Json(name = "viewability")
    public String viewability;
    @Json(name = "embeddable")
    public Boolean embeddable;
    @Json(name = "publicDomain")
    public Boolean publicDomain;
    @Json(name = "textToSpeechPermission")
    public String textToSpeechPermission;
    @Json(name = "epub")
    public Epub epub;
    @Json(name = "pdf")
    public Pdf pdf;
    @Json(name = "webReaderLink")
    public String webReaderLink;
    @Json(name = "accessViewStatus")
    public String accessViewStatus;
    @Json(name = "quoteSharingAllowed")
    public Boolean quoteSharingAllowed;

}

File: app/src/main/java/com/karim/booksapp/data/models/pojos/AccessInfo.kt
package com.karim.booksapp.data.models.pojos

import com.squareup.moshi.Json

class AccessInfo {

    @Json(name = "country")
    var country: String? = null
    @Json(name = "viewability")
    var viewability: String? = null
    @Json(name = "embeddable")
    var embeddable: Boolean? = null
    @Json(name = "publicDomain")
    var publicDomain: Boolean? = null
    @Json(name = "textToSpeechPermission")
    var textToSpeechPermission: String? = null
    @Json(name = "epub")
    var epub: Epub? = null
    @Json(name = "pdf")
    var pdf: Pdf? = null
    @Json(name = "webReaderLink")
    var webReaderLink: String? = null
    @Json(name = "accessViewStatus")
    var accessViewStatus: String? = null
    @Json(name = "quoteSharingAllowed")
    var quoteSharingAllowed: Boolean? = null

}
